scopes/lanes/diff/lane-diff-generator.ts

```diff
@@ -104,4 +133,38 @@ export class LaneDiffGenerator {
     const toLaneName = values.length === 2 ? values[1] : values[0];
     return { fromLaneName, toLaneName };
   }
+
+  private async getDefaultLaneData(ids: BitId[]): Promise<LaneData> {
+    const laneData: LaneData = {
+      name: DEFAULT_LANE,
+      remote: null,
+      components: [],
+      isMerged: null,
+    };
+    for await (const id of ids) {
```

Comment:
I'd suggest using `map` instead of `for` loop, and `Promise.all` to run them all concurrently.